pkg/discoveredclusters/discovered_cluster.go

```diff
@@ -0,0 +1,119 @@
+package discoveredclusters
```

Comment:
Depending on which of the PRs gets merged first, it might be worthwhile to have a single package hierarchy (i.e. `cloudsources/discoveredclusters` for that one).